frontend/src/app/pages/exercises/exercise/shared/trainer-map-editor/trainer-map-editor.component.html

```diff
@@ -167,7 +167,16 @@ <h5>Fahrzeuge</h5>
                     </div>
                 </li>
                 <li class="list-group-item">
-                    <h5>Bilder</h5>
+                    <h5>
```

Comment:
Maybe add a text similar to the transfer overview when no image template exists. It looks a bit weird right now, in my opinion.